graphql/src/main/java/com/linecorp/armeria/server/graphql/GraphqlErrorsHandlers.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.graphql;
+
+import java.util.List;
+
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+
+import graphql.ExecutionResult;
+import graphql.ExecutionResultImpl;
+import graphql.GraphQLError;
+import graphql.GraphqlErrorException;
+import graphql.validation.ValidationError;
+
+final class GraphqlErrorsHandlers {
+
+    static final GraphqlErrorsHandler defaultErrorsHandler =
```

Comment:
`static final` variables are conventionally underscore separated, uppercased.

```suggestion
    static final GraphqlErrorsHandler DEFAULT_ERRORS_HANDLER =
```

Reply:
@minwoox pointed out this doesn't necessarily have to be treated as a constant. Let's leave as is 😄

Reply:
@jrhee17 Got it!